lib/child-writer/writer-proxy.js

```diff
@@ -157,6 +158,13 @@ return isElevated().then((elevated) => {
       message: 'User did not grant permission.'
     }, () => {
       process.exit(EXIT_CODES.CANCELLED);
+    }).catch({
+      message: 'No polkit authentication agent found.'
```

Comment:
Is there any way that this message (and the other message a few lines further up) could be unit-tested, so that this code doesn't break if `sudoPrompt` should happen to change its error strings?